apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/InGax.cs

```diff
@@ -1,15 +1,16 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Google.Api.Gax
 {
     public sealed class BatchingSettings
     {
-        long? ElementCountThreshold { get; set; }
-        long? RequestByteThreshold { get; set; }
-        TimeSpan? DelayThreshold { get; set; }
+        public long? ElementCountThreshold { get; set; }
```

Comment:
Is the intention that these can be changed at any time? There's no element of thread safety / atomicity / volatility here.